lib/ex_unit/lib/ex_unit/capture_log.ex

```diff
@@ -37,7 +37,14 @@ defmodule ExUnit.CaptureLog do
   Returns the binary which is the captured output.
 
   This function mutes the `:console` backend
-  and captures any log messages sent to Logger.
+  and captures any log messages sent to Logger from the calling
+  processes. It is possible to ensure explicit log messages from other
+  processes are captured by waiting for their exit or monitor
+  signal. This does not guarantee to capture VM generated log messages
+  when a process spawned using a `spawn` BIF does not catch an exception.
```

Comment:
I've definitely heard the term BIF used in erlang docs, but don't think I've encountered it in the Elixir docs so I've assumed it wasn't really a term used by Elixir.  Should this be `Kernel.spawn` or similar instead?